CONTRIBUTING.md

```diff
@@ -1,32 +1,228 @@
-# Contributing
+# What The Hack - Contribution Guide
 
-The What The Hack project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
+What The Hack is all about being "for the people, by the people". This repo was originally created to share real-world hackathons that Microsoft employees have hosted with their customers. Here are our core principles:
```

Comment:
Would add after "...hosted with their customers." ....

It has since grown to be used as learned tools for anyone, anywhere, and follows the same core principles:

Reply:
Done... with slight modification.

"It has since grown to be a learning tool for anyone, anywhere, and follows these core principles:"